manifests/dev/vmipreset-resource.yaml.in

```diff
@@ -1 +1 @@
-{{index .GeneratedManifests "vmipreset-resource.yaml"}}
+{{index .GeneratedManifests "vmipreset-resource.yaml" -}}
```

Comment:
what's up with the - added here?

Reply:
golang templates will add an extra newline otherwise. Not really sure what to do otherwise.

Reply:
Found  fix for that. I let sed remove empty lines at the end of each file.